lms/djangoapps/notification_prefs/views.py

```diff
@@ -95,6 +96,8 @@ def enable_notifications(user):
     Enable notifications for a user.
     Currently only used for daily forum digests.
     """
+    # Calling UserPreference directly because this method is called from a couple of places,
+    # and it is not clear that user is always the user initiating the request.
     UserPreference.objects.get_or_create(
```

Comment:
I'm going to leave the 2 places in this file where UserPreference.objects.get_or_create is used alone. It's not obvious to me that the requesting user is always the same as the user being operated on (in particular in the second case, on line 190).